tools/pyorac/swath.py

```diff
@@ -582,15 +579,16 @@ def __getitem__(self, name):
 
         return np.ma.masked_invalid(data)
 
-
     # -------------------------------------------------------------------
     # Masking functions
     # -------------------------------------------------------------------
-    def mask_cloud(self, allowing=[]):
+    def mask_cloud(self, allowing=None):
         """Bool array that is True where the aerosol cloud flag is non-zero.
 
         The allowing argument can be used to exempt a list of members of the
         orac.CLDFLAG bitmask."""
+        if allowing is None:
```

Comment:
Probably need a second clause that listifies allowing if you pass a single string.

Reply:
True. fixed!